Cargo.toml

```diff
@@ -35,6 +35,7 @@ branch = "next"
 # For the examples
 tokio = { version = "1.4.0", features = ["rt-multi-thread"] }
 futures = { version = "0.3.13", default-features = false }
+poise = { path = ".", features = ["collector"] }
```

Comment:
I couldn't get the examples to compile without it- not sure if its something with my cargo setup or what, but because the serenity collectors feature isn't default, this was causing errors (using command `cargo run --example serenity_framework_usage`)

Reply:
Oh I see. You could have gotten it to work by passing `--features collector`, as mentioned in the examples README and crate root docs. But your change is nice because it means you don't need to explicitly pass `--features collector` anymore